test/parallel/test-http-server-unconsume-consume.js

```diff
@@ -4,8 +4,8 @@ const http = require('http');
 
 const testServer = http.createServer(common.mustNotCall());
 testServer.on('connect', common.mustCall((req, socket, head) => {
-  socket.write('HTTP/1.1 200 Connection Established' + '\r\n' +
-      'Proxy-agent: Node-Proxy' + '\r\n' +
+  socket.write('HTTP/1.1 200 Connection Established\r\n' +
+      'Proxy-agent: Node-Proxy\r\n' +
```

Comment:
Yes, and that's why it didn't complain about the next `'\r\n'`.

I think we're both saying the same thing -- did I misunderstand the question?